src/components/TableWithSearchFilters/index.tsx

```diff
@@ -111,6 +111,21 @@ const TableWithSearchFilters = (props: TableWithSearchFiltersProps) => {
     dispatchSearchRequest(activeLocale, search, searchSortOrder);
   }, [activeLocale, dispatchSearchRequest, getSearchPayload, searchSortOrder]);
 
+  // reset filters when extraTableFilters change
+  useEffect(() => {
+    setFilters((filters) => {
+      const newFilters = { ...filters };
+      Object.keys(newFilters)?.forEach((key) => {
+        const filter = {
+          ...newFilters[key],
+          values: [],
+        };
+        newFilters[key] = filter;
+      });
+      return newFilters;
+    });
+  }, [extraTableFilters]);
+
```

Comment:
This works and I think we should be resetting the filters any time the extra filters change anyways, but let me know if this is problematic for any reason.

Reply:
Looks good to me! Nice work!